frontend/shared/src/__tests__/utils/i18n/create-translations.ts

```diff
@@ -0,0 +1,18 @@
+// cannot import because build doesn't understand '@frontend/shared/src/i18n/i18n'
+// while testcase doesn't understand 'shared/i18n/i18n'
+// import { DEFAULT_LANGUAGE, Language } from '@frontend/shared/src/i18n/i18n';
```

Comment:
No need to do this, `@frontend/shared/src/` should work for both the build and testcafe run. I think that the problem is that if this file is in the shared project so it doesnt have refenrence to itself with `@frontend/shared`. So just use relative path `import { DEFAULT_LANGUAGE, Language } from '../../../i18n/i18n'`